helm/defectdojo/templates/celery-beat-deployment.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ include "defectdojo.chart" . }}
+{{- if .Values.extraLabels }}
+{{ .Values.extraLabels | toYaml | trim | indent 4 }}
+{{- end -}}
```

Comment:
The trailing `-` here will drag the line below onto this line, resulting in the Helm linter error raised [here](https://github.com/DefectDojo/django-DefectDojo/actions/runs/11442656501/job/32089722595?pr=11108). You can see it in action [here in the Helm Plaground](https://helm-playground.com/#t=LQhQG92ACASAzArgG2QOQIYFsCm0BcAvNAEQBGyGAFidMAL72gYAOAlgGo4BOAzmwHsAdvmisWvAPQA3AIygA1myEATUQBEcLZAICeuIQBdQuQxhUYz%2BUNGhDsOUZDhJUmXNEbAAxjmQ9dYDIcSxtoSmDkXmtbWxUceBxvQxUBACsBADoBbgBzSW8BLBZhHCNRX39uXTC4hKSU9Kyc-N5EMkLi0vLoYNDYsRYWTIV2niEcQxxeTMFJZV4zIV8ncGhMgCU-EN4cTPc8RlrB4dHg7gmpmbmsDHtcnBUg3VX1rf8MXcyAZR5pNl8niYkBgbHg6w4GGQiGmmRwAA9DNwMAAZDCRXhAiBrTKQ6GwhFI1HovyYgA%2B0EMAgAmthkNAKUi2FgGdBlPEjNAACxYkHQMoqOhHXgsJIxaDcLTIAGfV64qEwmaVAKZPqGTKS7QyzFHWx8sEQhWwyX-fjCAASbEWOV0KOZbEMWNsJqtgiElut1TtWAdcrxio1OFNbo9VK99sduugfIFQKAA&v=LQhQEsBcFMFsGcBcACA2gIgGbgE70ugDTLrzQDGA9gHYAm6AuqNAB6Q4CGAMhwEbQAbJKGTJeAjgAsU4qSLETJAJhmKlQA). This will likely cause errors in each of the other template files as well.

```suggestion
{{- end }}
```

![Screenshot 2024-10-25 at 17 28 44](https://github.com/user-attachments/assets/21d6b210-4923-42f7-8cfd-890bf98d2b04)

Reply:
Okay, so i reviewed my code and found it is not uniform with the existing code. 
I rewrote a big part of it and fixed what you found. 
Thanks for the review !